templates/uchiwa_config.json.j2

```diff
@@ -3,7 +3,7 @@
    {
        "name": "{{ uchiwa_dc_name }}",
        "host": "{{ sensu_api_host }}",
-       "ssl": {{ sensu_api_ssl }},
+       "ssl": {{ sensu_api_ssl | to_nice_json }},
```

Comment:
So, sadly, I think this change is resulting in:
```
{"timestamp":"2018-01-29T12:41:38.218583372Z","level":"fatal","message":"Error decoding file /etc/sensu/uchiwa.json: json: cannot unmarshal string into Go struct field SensuConfig.Ssl of type bool"}
``` 

We might want to look at replacing this with `| lower` instead, the issue being that this quotes the boolean, where as Uchiwa want's it as a lowercase unquoted boolean.